src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,20 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
+    private static final int SIZE = 6;
```

Comment:
Since you're not using any static methods, consider removing the 'static' modifier from the SIZE constant.